addons/tooltips/addon.json

```diff
@@ -0,0 +1,119 @@
+{
+  "name": "Tooltips",
+  "description": "Adds tooltips for projects, users, studios, forum topics, and forum posts. Uses the Scratch API and ScratchDB.",
```

Comment:
I think both the name and the description should mention that this adds tooltips to links.

Reply:
"Link tooltips" will be too weird. How about "Hovercards?" I think "Tooltips" is enough given the additional context on the description (will do later)

Reply:
I have changed it into "Hovercards". Hope it is adequate now.